Mathlib/Analysis/Meromorphic/NormalFormAt.lean

```diff
@@ -373,3 +374,271 @@ theorem meromorphicNFAt_toMeromorphicNFAt :
   mpr hf := by
     rw [hf]
     exact meromorphicNFAt_toMeromorphicNFAt
+
+/--
+If `f` is meromorphic in normal form, then so is its inverse.
+-/
+theorem MeromorphicNFAt.inv {f : 𝕜 → 𝕜} (hf : MeromorphicNFAt f x) :
+    MeromorphicNFAt f⁻¹ x := by
+  rcases hf with h | ⟨n, g, h₁, h₂, h₃⟩
+  · left
+    filter_upwards [h] with x hx
+    simp [hx]
+  · right
+    use -n, g⁻¹, h₁.inv h₂, (by simp_all)
+    filter_upwards [h₃] with y hy
+    simp only [Pi.inv_apply, hy, Pi.smul_apply', Pi.pow_apply, smul_eq_mul, mul_inv_rev, zpow_neg]
+    ring
+
+/--
+A function to 𝕜 is meromorphic in normal form at a point iff its inverse is.
+-/
+theorem meromorphicNFAt_iff_meromorphicNFAt_inv {f : 𝕜 → 𝕜} :
+    MeromorphicNFAt f x ↔ MeromorphicNFAt f⁻¹ x := by
+  constructor
+  · exact MeromorphicNFAt.inv
+  · nth_rw 2 [← inv_inv f]
+    exact MeromorphicNFAt.inv
```

Comment:
```suggestion
theorem meromorphicNFAt_inv {f : 𝕜 → 𝕜} : MeromorphicNFAt f⁻¹ x ↔ MeromorphicNFAt f x where
  mp := .inv
  mpr hf := by simpa using hf.inv
```
and consider marking it simp

Reply:
Applied, but needed to adjust a little to make it compile.